rllib/utils/replay_buffers/replay_buffer.py

```diff
@@ -130,21 +186,23 @@ def __len__(self) -> int:
     def add(self, batch: SampleBatchType, **kwargs) -> None:
         """Adds a batch of experiences to this buffer.
 
-        Also splits experiences into chunks of timesteps, sequences
-        or episodes, depending on self._storage_unit. Calls
-        self._add_single_batch.
+        Splits batch into chunks of timesteps, sequences or episodes, depending on
+        `self._storage_unit`. Calls `self._add_single_batch` to add resulting slices
```

Comment:
Btw, should we make `self._storage_unit` public? I feel like this is such an important setting in any buffer that the user should have a way to access this information. We can simply make this a public property.